packages/cli/index.ts

```diff
@@ -182,10 +183,16 @@ const main = defineCommand({
               cwd: templateDir,
               dot: true,
               onlyFiles: true,
+              ignore: ['node_modules', '.git'], // always ignore node_modules and .git
```

Comment:
I'm not sure if this affects fast-glob, since its default value is an empty one. Therefore, it's always a safe bet to include ignoring `node_modules` and `.git` directories in this context.

Reply:
makes sense, thank you so much